src/main/resources/db/schema/SL-1471-history-of-changes-listing.yaml

```diff
@@ -0,0 +1,121 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-hibernate-sequence
+      author: kpakur
+      comment: Without this sequence hibernate complains and application would not start.
+      failOnError: false
+      changes:
+        - createSequence:
+            schemaName: public
+            sequenceName: hibernate_sequence
+  - changeSet:
+      id: add-revinfo-table
+      author: kpakur
+      changes:
+        - createTable:
+            tableName: revinfo
+            columns:
+                - column:
+                    constraints:
+                      primaryKey: true
+                    name: rev
+                    type: int
+                - column:
+                    name: revtstmp
+                    type: bigint
+  - changeSet:
+      id: create-hearingpart-history-table
+      author: kpakur
+      changes:
+        - createTable:
+            tableName: hearing_part_aud
+            columns:
+            #audit columns
+            - column:
+                name: rev
+                type: int
+                constraints:
+                  nullable: false
+                  foreignKeyName: fk_hearing_part_aud_revinfo
+                  references: revinfo(rev)
+            - column:
+                name: revtype
+                type: smallint
+            #hearingpart columns
+            - column:
+                name: id
+                type: UUID
+                constraints:
+                  nullable: false
+            - column:
+                name: case_number
+                type: VARCHAR(200)
+            - column:
+                name: case_title
+                type: VARCHAR(200)
+            - column:
+                name: case_type
+                type: VARCHAR(100)
+            - column:
+                name: hearing_type
```

Comment:
So far code looks good. I just noticed some inconsistency with US. In the requirements from SL-1349, we've had to add hearing_type table (with code and description) but there was no information about connecting hearing type to listing hearing part and here we have hearing_type column that is not connected with new table.

Reply:
discussed